docs-hub/lychee.toml

```diff
@@ -0,0 +1,26 @@
+# Enable link caching. This can be helpful to avoid checking the same links on
+# multiple runs.
+cache = true
+
+# Discard all cached requests older than this duration.
+max_cache_age = "1d"
+
+# Comma-separated list of accepted status codes for valid links.
+accept = ["200", "206", "405"]
+
+# Exclude URLs and mail addresses from checking (supports regex).
+exclude = [
+    'localhost:',
+    '^http://[1-9]',
+    'crates\\.io',
+    'faucet-beta-5.fuel.network',
+    '-indexer.fuel-network',
+    'fuellabs.github.io/block-explorer-v2',
+    'adobe\\.com',
+    'scripts.sil.org',
```

Comment:
why are we excluding these ?

Reply:
I copied all of these from the existing MLC config. I'll try removing, maybe we don't need some/all of them.

Reply:
Here are the only failing links:
```
[docs/src/connecting/external-node.md]:
✗ [404] https://fuellabs.github.io/block-explorer-v2 | Failed: Network error: Not Found
✗ [403] https://faucet-beta-5.fuel.network/ | Failed: Network error: Forbidden
```
The URLs are working, though. I'll leave those ignored, and remove all other rules.